plugins/modules/aci_fabric_pod_remote_pool.py

```diff
@@ -0,0 +1,288 @@
+                }
+            }
+        }
+    ]
+proposed:
+  description: The assembled configuration from the user-provided parameters
+  returned: info
+  type: dict
+  sample:
+    {
+        "fvTenant": {
+            "attributes": {
+                "descr": "Production environment",
+                "name": "production"
+            }
+        }
+    }
+filter_string:
+  description: The filter string used for the request
+  returned: failure or debug
+  type: str
+  sample: ?rsp-prop-include=config-only
+method:
+  description: The HTTP method used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: POST
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec, aci_owner_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(aci_owner_spec())
+    argument_spec.update(
+        pod_id=dict(type="int", aliases=["pod"]),
+        description=dict(type="str", aliases=["desc"]),
+        remote_id=dict(type="int", aliases=["id"]),
+        name_alias=dict(type="str"),
+        remote_pool=dict(type="str", aliases=["pool"]),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["pod_id", "remote_id"]],
+            ["state", "present", ["pod_id", "remote_id"]],
+        ],
+    )
+
+    aci = ACIModule(module)
+
+    pod_id = module.params.get("pod_id")
+    description = module.params.get("description")
+    remote_id = module.params.get("remote_id")
+    name_alias = module.params.get("name_alias")
+    remote_pool = module.params.get("remote_pool")
+    state = module.params.get("state")
+
+    if pod_id is not None and int(pod_id) not in range(1, 254):
+        aci.fail_json(msg="Pod ID: {0} is invalid; it must be in the range of 1 to 254.".format(pod_id))
+
+    aci.construct_url(
+        root_class=dict(
+            aci_class="fabricSetupP",
+            aci_rn="controller/setuppol/setupp-{0}".format(pod_id),
+            module_object=pod_id,
+            target_filter={"podId": pod_id},
+        ),
+        subclass_1=dict(
+            aci_class="fabricExtSetupP", aci_rn="extsetupp-{0}".format(remote_id), module_object=remote_id, target_filter={"extPoolId": remote_id}
```

Comment:
put a comma behind and run black to format similar to root class above
```suggestion
            aci_class="fabricExtSetupP", aci_rn="extsetupp-{0}".format(remote_id), module_object=remote_id, target_filter={"extPoolId": remote_id},
```